department-of-veterans-affairs/caseflow
# Pamatya/appeals 39105

Resolves [APPEALS-39105](https://jira.devops.va.gov/browse/APPEALS-39105)

# Description
Please explain the changes you made here.
In AddClaimantConfirmationModal, added a condition to check if benefit type is VHA then we display 'No Known POA' instead of 'Intake does not have a form 21-22' in model. 

## Acceptance Criteria
- [ ] IF benefit_type = 'vha' then 
- [ ] for relationShip: Child, Spouse, Healthcare Provider, Other
Prompt: 'Do you have a VA Form 21-22 for this claimant?'
No = No known POA
Yes
Name selected from list = POA information
Name not listed = No recognized POA
Attorney (previously or currently)
There is no prompt for 'Do you have a VA Form 21-22 for this claimant?'
No known POA

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2. Login as a ACBAUERVVHAH or any user that can do Intake and also select VHA benefit type.
3. Do HLR or SC intake
4. select benefit type== 'vha', 
5. Add Claimant -> Relationship to the veteran -> Child -> fill all required fields, -> Do you have a VA Form 21-22 for this claimant? -> no 
6. Click "Continue to next step" 
7. In the confirmation modal you will see -> 'No known POA'
8.  Similarly intake input for all other condition in the AC
9. 



# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/122557351/ab612c11-07af-4e1f-b6b8-85ddf2526aa7)

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
